api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/organizations_controller.rb

```diff
@@ -25,30 +25,20 @@ module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::OrganizationsContro
 
     # Overwrite check support authorization for index action only.
 
-    # Workaround because using only and if are not possible (it checks to see if either satisfy, not both.)
-    # https://github.com/rails/rails/issues/9703#issuecomment-223574827
-    skip_before_action :block_support_users, if: -> { support_org_external_id && action_name == 'index'}
-    before_filter :support_enabled?, if: -> { support_org_external_id && action_name == 'index'}
-
-    skip_before_action :block_support_users, only: [:show]
+    skip_before_action :block_support_users, only: [:show, :support_search]
     before_filter :authorize_support_user_organization, only: [:show]
+    before_filter :valid_support_search, only: [:support_search]
```

Comment:
If it's just one action you don't really need the `before_action`.

```ruby
def support_search
  return head :forbidden unless valid_support_seach
  ...
end
```
It's also probably easier to read?

Reply:
sure!